modules/workbench/variables.tf

```diff
@@ -55,3 +55,44 @@ variable "notification_channels" {
     }
   }]
 }
+
+#
+# Egress
+#
+variable tiers {
+  description = "List of tiers"
+  type        = list(string)
+  default     = ["registered"]
+}
+
+variable sumologic_egress_thresholds {
+  description = "Configuration values for egress search content in SumoLogic. The name (key) describes simply the tier name and config: "
+  type        = map(map(any))
+  default = {
+    registered_tier_60sec_50mib = {
```

Comment:
nit: consider renaming 60sec -> 1m, 600sec -> 10m, 3600sec -> 1h?

Reply:
Done. I used sec before because `egress_window_sec` uses that.